pkg/kubectl/resource/builder.go

```diff
@@ -169,6 +170,13 @@ func (b *Builder) Local(mapperFunc ClientMapperFunc) *Builder {
 	return b
 }
 
+// TabularResponse sets the header "Accept: application/json;as=Table;v=v1alpha1;g=meta.k8s.io"
+// as part of a request to a resource endpoint
+func (b *Builder) TabularResponse(tabular bool) *Builder {
```

Comment:
This isn't going to work well with unstructured builders.  I don't know that the builder is responsible for knowing about tables.  At some point we're going to have commands that deal with generic metadata, and those shouldn't get their own methods.  I think this is a lower level detail that needs to be handled more cleanly.